swagger/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 The Tekton Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM swaggerapi/swagger-ui
+ENV SWAGGER_JSON=/foo/tekton-dashboard-api.yaml
+COPY tekton-dashboard-api.yaml /foo/
+
```

Comment:
Let's remove the whitespace, is /foo/ legit?

Reply:
Yes, the /foo is legit for the base swagger-ui image